foundation.dd

```diff
@@ -0,0 +1,58 @@
+Ddoc
+
+$(D_S $(TITLE),
+
+$(H4 The D Language Foundation is an organization $(B devoted) to advancing
+open source technology related to the $(B D programming language).)
+
+$(H3 History)
+
+$(P The D programming language was started by Walter Bright, as a 'better C++'
+and has grown slowly, but steadily, for more than 15 years. Recent improvements
+to the D development process have accelerated development of the language and
+shown the need for a governing organization that overlooks and arranges the
+future development of the D programming language.)
+
+$(H3 Current projects)
+
+The D foundation
+
+$(UL
+
+    $(LI is providing the various resources for the D community
+        (e.g. this website, dmd downloads, the D tour and a lot more))
+    $(LI is an accepted
+        $(LINK2 https://summerofcode.withgoogle.com/organizations/5078256051027968/,
```

Comment:
This is an antipattern, doing so defeats half the point of HTTPS (as the HTTP redirect can be intercepted). We should go the other way.